microsoft/botframework-solutions
# [Enterprise Bot Generator][TypeScript] Create a npmignore for generators

## Description
- Add the .npmignore file
- Change the name of the` .gitignore `extension to avoid the error that the`.gitignore` disappear when the package was published

## Related Issue
Fix  #678 

## Testing Steps
1. Go to` AI\templates\Enterprise-Template\src\typescript\generator-botbuilder-enterprise`.
2. Check that the `.npmignore` was created on this folder.
3. Go to `AI\templates\Enterprise-Template\src\typescript\generator-botbuilder-enterprise\generators\app\templates\enterprise-bot`
4. Check that the name of `.gitignore` extension was changed on this folder.

## Checklist
~- [ ] I have commented my code, particularly in hard-to-understand areas~
~- [ ] I have added or updated the appropriate unit tests~
~- [ ] I have tested any new/updated dialogs using Speech in the emulator to ensure the [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) property is set to enable a high quality speech-first experience and the appropriate [InputHints](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) are set correctly.~ 
~- [ ] I have updated related documentation~

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
~- [ ] A duplicate issue is filed to track future  work.~

If you have updated responses or `.lu` files:
~- [ ] All languages have been updated~
~- [ ] You have tested deployment with your new models~
